SonarQube.Common/ProcessRunner.cs

```diff
@@ -61,6 +61,11 @@ public bool Execute(ProcessRunnerArguments runnerArgs)
                 WorkingDirectory = runnerArgs.WorkingDirectory
             };
 
+
+            //string tempString = runnerArgs.CmdLineArgs != null ? String.Join(" ", runnerArgs.CmdLineArgs) : "";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=csharpsquid%3AS125)